src/Components/SliderComponent.js

```diff
@@ -0,0 +1,53 @@
+import ClassCard from "./ClassCard";
+import styled from "styled-components";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/swiper.scss";
+import "swiper/components/navigation/navigation.scss";
+
+const SliderComponent = ({ dataList, showingSlidesCount }) => {
+  return (
+    <SliderWrapper>
+      <Swiper slidesPerView={showingSlidesCount || 4}>
+        {!!dataList.length &&
+          dataList.map((data) => {
```

Comment:
```suggestion
        {dataList?.map((data) => {
```
optional chaining을 사용할 수도 있습니다